<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CETServer</class>
 <widget class="QMainWindow" name="CETServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CETServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QToolButton" name="openDatabaseButton">
      <property name="text">
       <string>Open Database</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="startServerButton">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="stopServerButton">
      <property name="text">
       <string>Stop Server</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="statusText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startServerButton</sender>
   <signal>clicked()</signal>
   <receiver>CETServer</receiver>
   <slot>startServerButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopServerButton</sender>
   <signal>clicked()</signal>
   <receiver>CETServer</receiver>
   <slot>stopServerButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openDatabaseButton</sender>
   <signal>clicked()</signal>
   <receiver>CETServer</receiver>
   <slot>openDatabaseButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startServerButtonHandler()</slot>
  <slot>stopServerButtonHandler()</slot>
  <slot>openDatabaseButtonHandler()</slot>
 </slots>
</ui>
